first-timers.md

```diff
@@ -7,18 +7,17 @@ To make changes to any page on http://sendgrid.com/docs, follow these steps:
 
 1. Navigate to the page you would like to edit and click **View and Edit** in the upper right-hand corner.
 
-<img src="https://github.com/Whatthefoxsays/docs/blob/github-cheatsheet/source/img/docs_view_and_edit.png" width="800" height="500">
+<img src="https://github.com/sendgrid/docs/blob/develop/source/img/docs_view_and_edit.png" width="800" height="500">
 
```

Comment:
What if we used relative paths instead?
```<img src="source/img/docs_view_and_edit.png" width="800" height="500">```
This would keep the image there even when the repo is forked and the images get deleted from the original repo.